components/Rooms/index.js

```diff
@@ -0,0 +1,10 @@
+export default function Rooms() {
+  return (
+    <>
+      <label htmlFor="rooms">Rooms</label>
+      <select name="rooms" id="rooms">
```

Comment:
If there is only one option, I guess a select is not necessary jet :)

Reply:
Thanks for the hint. I replaced the `select` with a `span` for now and implement the select in a later US